.github/workflows/test.yml

```diff
@@ -379,6 +379,13 @@ jobs:
     - uses: actions/checkout@v4
       with:
         fetch-depth: 0
+    - uses: actions/checkout@v4
```

Comment:
I think it will be complete once you add a `name` field to this `step` that indicates what is being checked out. 👍🏻